patching/plugin-commands-patching/src/getPatchedDependency.ts

```diff
@@ -76,7 +75,10 @@ export async function getVersionsFromLockfile (dep: ParseWantedDependencyResult,
   if (!lockfile) {
     throw new PnpmError(
       'PATCH_NO_LOCKFILE',
-      `No ${WANTED_LOCKFILE} found: Cannot patch without a lockfile`
+      'No node_modules/.modules.yaml found',
```

Comment:
This error message is not informative to a user. Something like `The modules directory is not ready for patching` will be better.